README.md

```diff
@@ -12,12 +12,28 @@ It is also a Stranger Things reference.
 Picture of Eleven
 -->
 
-## Install
+### Install
```

Comment:
Why change title level?

Reply:
The same is done in `fastify`. But I'll revert that. Should benchmarks stay here or should I move them down?

Reply:
I’ll leave it at your discretion.